cudf/tests/test_dummyarray.py

```diff
@@ -0,0 +1,92 @@
+
+import numpy as np
+from cudf.dummyarray import Array
+
+test_slices = [
+    slice(None, None, None),
+    slice(None, 3, None),
+    slice(3, None, None),
+    slice(None, None, 3),
+    slice(1, 5, None),
+    slice(1, None, 2),
+    slice(None, 5, 2),
+    slice(1, 5, 2),
+]
+
+test_indices = [
+    1, 4
+]
+
+
+def test_1d():
```

Comment:
These should probably be skipped if not on linux

Reply:
Non-linux is not an option for cudf currently.